pynumaflow/sink/async_sink.py

```diff
@@ -11,6 +11,8 @@
     SINK_SOCK_PATH,
     MAX_MESSAGE_SIZE,
 )
+from pynumaflow.info import info_types, info_server
```

Comment:
Why are we importing the whole file? Just import what we need. This is why it is usually recommended to have in the __init__ file of what you want to import

Reply:
since this is only used internally, we could avoid the `__init__` strict importing style.